modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -33,218 +45,373 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
+  uid: cca245a5-4bca-45fa-89c0-aa6af81a596c
+Spec:
+  actions:
+    afterCompletion:
+      deleteObjects: true
+    beforeEnable: {}
+  backup: false
+  clusters: <1>
+    - spoke1
+  enable: true <2>
+  managedPolicies: <3>
+    - talm-policy
+  preCaching: false
+  remediationStrategy: <4>
+    canaries: <5>
+        - spoke1
+    maxConcurrency: 2 <6>
+    timeout: 240
+  clusterLabelSelectors: <7>
+    - matchExpressions:
+      - key: label1
+      operator: In
+      values:
+        - value1a
+        - value1b
+  batchTimeoutAction: <8>
+status: <9>
+    clusters:
+      - name: spoke1
+        state: complete
+      - currentPolicy:
+          name: talm-policy
+          status: NonCompliant
+        name: spoke2
+        state: timedout
+    computedMaxConcurrency: 2
+    conditions:
+      - lastTransitionTime: '2022-11-18T16:27:15Z'
+        message: All selected clusters are valid
+        reason: ClusterSelectionCompleted
+        status: 'True'
+        type: ClustersSelected <10>
+      - lastTransitionTime: '2022-11-18T16:27:15Z'
+        message: Completed validation
+        reason: ValidationCompleted
+        status: 'True'
+        type: Validated <11>
+      - lastTransitionTime: '2022-11-18T16:37:16Z'
+        message: The cluster backup is in progress
+        reason: NotStarted
+        status: 'False'
+        type: Progressing
+      - lastTransitionTime: '2022-11-18T16:37:16Z'
+        message: Policy remediation took too long
+        reason: TimedOut
+        status: 'False'
+        type: Succeeded
+    managedPoliciesForUpgrade:
+      - name: talm-policy
+        namespace: talm-namespace
+    managedPoliciesNs:
+      talm-policy: talm-namespace
+    remediationPlan:
+      - - spoke1
+        - spoke2
+    status:
+      currentBatch: 1
+      currentBatchRemediationProgress:
+        spoke1:
+          state: Completed
+        spoke2:
+          policyIndex: 0
+          state: InProgress
+      currentBatchStartedAt: '2022-11-18T16:27:16Z'
+      startedAt: '2022-11-18T16:27:15Z'
+
+----
+<1> Defines the list of clusters to update.
+<2> The `enable` field is set to `true`.
+<3> Lists the user-defined set of policies to remediate.
+<4> Defines the specifics of the cluster updates.
+<5> Defines the clusters for canary updates.
+<6> Defines the maximum number of concurrent updates in a batch. The number of remediation batches is the number of canary clusters, plus the number of clusters, except the canary clusters, divided by the maxConcurrency value. The clusters that are already compliant with all the managed policies are excluded from the remediation plan.
+<7> Displays the parameters for selecting clusters.
+<8> Controls what happens if a batch times out. Possible values are `abort` or `continue`. If unspecified, the default is `continue`.
+<9> Displays information about the status of the updates.
+<10> The status for `clustersSelected` is `true`.
+<11> The status for `Validated` is `true`.
```

Comment:
I see better notes for these two in later examples. Move them here?

Reply:
done